opendrive2lanelet2/convertor/opendrive2lanelet2convertor.py

```diff
@@ -103,12 +126,43 @@ def process_vertices(self, vertices, relation_id, bound_id):
                 self.nodes.append(node.create_xml_node_object())
                 self.all_nodes.append(node)
         return nodes
+    # calculate area between multiple curves
+    def area_between_curve(self,c1,c2):
+
+        c1_fit = np.polyfit(c1[0],c1[1],4)
+        c2_fit = np.polyfit(c2[0],c2[1],4)
+
+        c1_1d_fun = np.poly1d(c1_fit)
+        c2_1d_fun = np.poly1d(c2_fit)
+        
+        n = np.polyint((c2_1d_fun - c1_1d_fun))
+        return n
+    # check for way duplication in the entire map
+    def check_way_duplication(self,nodes,way):
+        for k in self.all_ways:
+            test_x = [j.local_x for j in k.nodes]
+            test_y = [j.local_y for j in k.nodes]
+            x = [j.local_x for j in nodes]
+            y = [j.local_y for j in nodes]
+
+            intersection_test_x = list(set(test_x) & set(x))
+            intersection_test_y = list(set(test_y) & set(y))
+
+            if(len(intersection_test_x) > 20 and len(intersection_test_y) > 30):
```

Comment:
Let's leave a comment on how these numbers were derived whether if it is adjustable or not. 
And maybe give them a name as a variable